### ISSUE TITLE
Suggestion: chiptunes drum set on channel 10

### ISSUE THREAD

**Issue by bromagosa** [DerSpanischGamer/ay-3-8910-midi#1]
I've just received my Amadeus board. Thank you!

After having flashed it with the MIDI firmware, I played a bit with it and began trying out a few modifications to the source. I'm now wondering whether it'd be hard to add a simple drum kit that listens to GM drum notes on channel 10.

I've found a couple of projects that synthesize drums on the AY-3-8910, like [Andy4495](https://github.com/Andy4495/AY3891x)'s library, but when I tried to run it on the Amadeus board I couldn't get it to play a sound for the life of me. Just in case you have a quick answer to this, here's how I'm initializing the chip according to your schematics:

```
// Be sure to use the correct pin numbers for your setup.
// Note that on ATmega328: 17=A3, 16=A2, 15=A1, 14=A0
//          DA7, DA6, DA5, DA4, DA3, DA2, DA1, DA0, BDIR, BC2, BC1
AY3891x psg( 7,   6,   5,   4,   3,   2,   9,   8,   A1,  AY3891x::NO_PIN,  A0);
```

Any help you can provide will be very appreciated! :)

**Comment by Andy4495** [comment]
My [AY3891x](https://github.com/Andy4495/AY3891x) library was written so that it is processor-independent. It uses the digitalWrite() commands instead of writing directly to processor registers to control the sound generator chip. This allows the code to work with a wide array of processors off the shelf, without having to modify the code for each one. In order to get this approach to work, the library needs all three of the bus control signals (BDIR, BC1, BC2) to be connected to the processor. Unfortunately, the Amadeus board (per the schematic in the user guide) ties the BC2 signal high, so this hardware configuration won't work with my library. I haven't looked closely at the board layout, but if it is possible to modify the board to connect the BC2 pin to an unused IO pin on the '328 chip (looks like A4 is unused), then you could use my library. 

TL;DR: The AY3891x library won't work with the hardware as it is currently designed.
